### DISCUSSION TITLE
How to obtain data for VAT invoice?

### DISCUSSION THREAD

**Discussion by lukaszApilo** [amzn/selling-partner-api-models#3775] [General]
Hi,

We are faced with the impossibility of effectively obtaining VAT invoice information for the buyer.
We need to know how we can obtain the information whether the buyer wants a VAT invoice and how to get the invoice data using your Selling Partner API? 
It is not possible to perform this using the `/orders/v0/orders/` resources, there is only `IsBusinessOrder` field and we presume that this is an indication of the customer's desire to receive an invoice but is not possible to obtain enough data to issue a VAT invoice.
How can we get such information using your Selling Partner API? The solution must work for the markets Poland, Germany, France, Italy, UK. 
When we fetch report `GET_FLAT_FILE_VAT_INVOICE_DATA_REPORT` via SP API the result is `You must be enrolled in Amazon’s VAT Calculation Service to get this report.`.

**Comment by stefnats** [comment]
Which data exactly do you need? 

Please note that you should have the appropriate [roles](https://developer-docs.amazon.com/sp-api/docs/roles-in-the-selling-partner-api) and also [access to PII information](https://developer-docs.amazon.com/sp-api/docs/register-as-a-public-developer#developer-profile-review-for-restricted-pii-roles) in case you need buyer name + address

**Comment by lukaszApilo** [comment]
Data we need to create a VAT invoice: clear information whether the user requests the issuance of a VAT document and then Company name, Tax ID number, company address, company phone/mail.

**Comment by domeglic** [comment]
I faced the same problem, I'm not sure how Amazon expects sellers to create an invoice without having access to the billing address of a customer. As you have seen you need to activate and use their VAT invoice calculation which will create these invoices instead. 

But we were advised by a consultant that this service often creates invoices with wrong amounts which results in more work to correct them and were told to instead just manually upload an invoice whenever a customer complains about needing an invoice with the correct address. Since most users use the same address as billing or don't care about the invoice.
